sunpy/instr/lyra.py

```diff
@@ -1,123 +1,581 @@
 from __future__ import absolute_import
+from __future__ import division
 
+import os.path
 import datetime
-import calendar
+from warnings import warn
+import copy
+import csv
+import urllib
 import sqlite3
+
 import numpy as np
-import os
+from astropy.io import fits
+import pandas
+
 from sunpy.time import parse_time
-import urllib
+from sunpy import config
+from sunpy.util.net import check_download_file
+from sunpy import lightcurve
 
-def download_lytaf_database(lytaf_dir=''):
-    """download the latest version of the Proba-2 pointing database from the Proba2 Science Center"""
-    #dl=sunpy.net.download.Downloader()
-    #dl.download('http://proba2.oma.be/lyra/data/lytaf/annotation_ppt.db',path=lytaf_dir)
-    url='http://proba2.oma.be/lyra/data/lytaf/annotation_ppt.db'
-    destination=os.path.join(lytaf_dir,'annotation_ppt.db')
-    urllib.urlretrieve(url,destination)
+LYTAF_REMOTE_PATH = "http://proba2.oma.be/lyra/data/lytaf/"
+LYTAF_PATH = config.get("downloads", "download_dir")
 
-    return
+def remove_lytaf_events_from_lightcurve(lc, artifacts=[],
+                                        return_artifacts=False,
+                                        lytaf_path=LYTAF_PATH,
+                                        force_use_local_lytaf=False):
+    """
+    Removes periods of LYRA artifacts defined in LYTAF from a LYRALightCurve.
```

Comment:
`LYRALightCurve` should be a link using the `<backtick>~sunpy.lightcurve.LYRALightCurve<backtick>`